drift/views/admin.py

```diff
@@ -0,0 +1,24 @@
+from flask import Blueprint, jsonify
```

Comment:
Again, unless I am misunderstanding the new admin work, I think this file needs to be drift/admin_views/v1.py instead.

Reply:
given your decision to not have version, then this is probably drift/admin_views/admin.py for the naming convention then?